tests/app/service/test_rest.py

```diff
@@ -888,30 +889,6 @@ def test_should_not_update_service_with_duplicate_name(notify_api, sample_servic
             assert "Duplicate service name '{}'".format(service_name) in json_resp['message']['name']
 
 
-@pytest.mark.skip(reason='Mislabelled for route removal, fails when unskipped')
```

Comment:
Functionality does not exist in code base.

Reply:
The functionality to use the `email_from` attribute does not exist. 

The error log `Duplicate service email_from` does not appear to be in the codebase any more.

Reply:
It actually hits `@service_blueprint.route('/<uuid:service_id>', methods=['POST'])`  and the method `update_service`

There's nothing in this route that says `email_from` needs to be unique. And in the Service model `email_from` is marked as `unique=False`.

So I don't think the functionality to make sure `email_from` is unique to a service exists.